src/server.ts

```diff
@@ -194,7 +194,7 @@ app.post("/bridgeCallAnswered", async (req, res) => {
   });
 
   const resp = new Response();
-  resp.add(speak);
+  // resp.add(speak);  //this introduces a 1-way voice condition - TODO - fix the root cause and reinstate the annct
```

Comment:
I would completely remove this line, and add a JIRA ticket or just a TODO comment with the work to be done

Reply:
wisdom !  thanks.

Reply:
SpeakSentence stuff removed, and ticket EDGE-1171 opened to replace it someday (taking bets ?)